CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/model/CellValue.java

```diff
@@ -23,6 +23,7 @@
 
     private final String originalText;
 
+    @SuppressWarnings("rawtypes") //raw type needed as comparable value can be string or integer
```

Comment:
@antares1470 does it help get rid of the warning if you change this to Comparable<Object> though that is what we are dealing with already?

Reply:
Sadly not in this case